src/vs/code/electron-sandbox/issue/issueReporterService.ts

```diff
@@ -278,6 +278,15 @@ export class IssueReporter extends Disposable {
 		}
 	}
 
+	private async getReporterStatus(extension: IssueReporterExtensionData): Promise<boolean[]> {
+		try {
+			const data = await this.issueMainService.$getReporterStatus(extension.id, extension.name);
+			return data;
+		} catch (e) {
```

Comment:
Would be nice to log this error even if it's just to the console of the issue reporter